src/main/java/ch/uzh/ifi/fs22/sel/group15/cozycave/server/entity/listings/Listing.java

```diff
@@ -93,6 +91,27 @@ public class Listing implements Cloneable {
     @JoinColumn(name = "publisher_id", nullable = false)
     private User publisher;
 
+    public Listing(UUID id, Date creationDate, String title,
```

Comment:
isn't this the same as `@AllArgsConstructor`?

Reply:
I needed it to create the tests, otherwise I couldn't create a listing for the application.

Reply:
so you needed a constructor not including rent? because I wouldn't create a constructor just for testing. you could just use the allargs one and pass null. this is better